openedx/core/djangoapps/theming/finders.py

```diff
@@ -0,0 +1,65 @@
+"""
+Static file finders for Django.
+https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-STATICFILES_FINDERS
+Yes, this interface is private and undocumented, but we need to access it anyway.
```

Comment:
I haven't asked anyone about making this interface supported. It might be worth doing that in the future, but that is additional work that is not necessary to get this pull request merged, and even if the interface becomes supported, that won't help us until we are on a version of Django that includes that supported interface.

As far as I know, other open source projects like Django-Pipeline and Django-Require simply use this interface in the same way, despite it being unsupported.